src/main/java/com/quantumsoft/qupathcloud/entities/DicomAttribute.java

```diff
@@ -17,55 +17,94 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-
 import java.util.Arrays;
 import java.util.Objects;
 
-public class DicomAttribute<T>{
-    private String vr;
-    @JsonProperty
-    private T[] Value;
+/**
+ * DicomAttribute class may contain any DICOM attribute.
```

Comment:
I would describe what a DICOM Attribute  or what the supported attibutes are..this is not adding any information that will help the client.